doc/REPLICATED.md

```diff
@@ -0,0 +1,79 @@
+# Installing Waffle Takeout
+
+Please submit a request on [https://takeout.waffle.io](https://takeout.waffle.io) for more information on pricing and to register for a 45 day trial.
+
+Getting Started guides and other Waffle.io documentation can be found at [here](https://github.com/waffleio/waffle.io/wiki).
+
+## Prerequisites
+
+* A dedicated host with Ubuntu 14.04 or greater installed
+* A Waffle Takeout license file
```

Comment:
can you add `(downloadable from [takeout.waffle.io](https://takeout.waffle.io))`